tests/components/heos/test_media_player.py

```diff
@@ -106,7 +102,7 @@ async def test_state_attributes(
     assert ATTR_INPUT_SOURCE not in state.attributes
     assert (
         state.attributes[ATTR_INPUT_SOURCE_LIST]
-        == hass.data[DOMAIN][DATA_SOURCE_MANAGER].source_list
+        == config_entry.runtime_data.source_manager.source_list
```

Comment:
These also should be removed in the long run. Im not blocking it for this PR as it will become huge with all these little additions